paasta_tools/cli/cmds/logs.py

```diff
@@ -744,6 +758,9 @@ def callback(
             clusters=clusters, components=components, callback=callback
         )
 
+        strip_cluster = True if len(clusters) < 2 else False
```

Comment:
nit: just `strip_cluster = len(clusters) < 3`? similar for `strip_instance`

Reply:
Removed `strip_cluster` and `strip_instance` altogether as it is unnecessary.